sklearn/preprocessing/tests/test_encoders.py

```diff
@@ -637,3 +637,72 @@ def test_categories(density, drop):
 @pytest.mark.parametrize('Encoder', [OneHotEncoder, OrdinalEncoder])
 def test_encoders_has_categorical_tags(Encoder):
     assert 'categorical' in Encoder()._get_tags()['X_types']
+
+
+@pytest.mark.parametrize('Encoder', [OneHotEncoder, OrdinalEncoder])
+@pytest.mark.parametrize('col, categories', [
+    ('int_col', [3, 1, 2]),
+    ('str_col', ['d', 'z', 'u']),
+    ('float_col', [1.0, 3.1, 2.3]),
+])
+def test_multiple_pandas_category_no_order(Encoder, col, categories):
+    # order pandas categories will warn for ordered categories
```

Comment:
Sorry @thomasjpfan but that test is confusing.

```suggestion
    # order pandas categories will warn for ordered categories
    # Make sure warning is raised when pandas category ordering isn't consistent with the encoder's
```

The test name also suggests that there is no order.